scripts/clean-dist.js

```diff
@@ -0,0 +1,3 @@
+(async () => {
```

Comment:
`clean-dist` and `remove-dist` could be done using `rm -rf dist/` i assume
no need to import packages for that.